<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PayListBelarusbank</class>
 <widget class="QDialog" name="PayListBelarusbank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>409</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>409</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>409</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Беларусбанк: платежный список</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Порядковый номер списка:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editNsp">
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Номер структурного подразделения:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editNotd"/>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Номер филиала:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editNfil"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Номер счета:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxAcc"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Фамилия, инициалы состовителя списка:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editFisp">
     <property name="maxLength">
      <number>45</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Назначение перечисления:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editNzp">
     <property name="maxLength">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Номер платежного поручения:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editNpp">
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Номер заключенного договора:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editContract">
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Подтвердить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
